### DISCUSSION TITLE
INTERNAL_SERVER_ERROR

### DISCUSSION THREAD

**Discussion by iamraphealc** [vercel/community#2800] [Help] [answered]
### Page to Investigate

https://rauto-api.vercel.app/listing

### Steps to Reproduce

![Screenshot (119)](https://github.com/vercel/community/assets/113741914/301d0fe9-548b-4f3e-811c-456fb975a443)
I get This Serverless Function has crashed. but this doesn't happen locally please i need you help

**Comment by smaeda-ks** [comment, marked as answer]
@Kingrashy12 quickly scanned your [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs) (please do this by yourself next time).

> [ERROR] [1686283203068] LAMBDA_RUNTIME Failed to post handler success response. Http response code: 413.

Sounds like you're hitting our max response body size limit here:
https://vercel.com/docs/concepts/limits/overview#serverless-function-payload-size-limit

Could you reduce your response body size?

**Reply by iamraphealc** [reply]
The response body is based on the application request
